common/src/WooCommerce.php

```diff
@@ -204,4 +191,28 @@ private function call_method($obj, $name) {
         }
         return @$method->invoke($obj);
     }
+
+    private function get_product_data(array $order_arr) {
+        $pf = new WC_Product_Factory();
+        $products = [];
+        foreach ($order_arr['line_items'] as $line_item) {
+            $product = $pf->get_product($line_item['product_id']);
+            if (!$product) {
+                continue;
+            }
+            $images = [];
+            foreach (get_attached_media('image', $product->get_id()) as $image) {
+                $images[] = wp_get_attachment_image_src($image->ID, 'full')[0];
+            }
```

Comment:
What's the point of getting all images if you only send the first one?

~~~
private function getProductImage(WP_Product $product) {
  foreach (get_attached_media('image', $product->get_id()) as $image) {
    return wp_get_attachment_image_src($image->ID, 'full');
  }
}
~~~